### PR TITLE
Update NPC2 for Quarterfinals

### PR DESCRIPTION
**Pull request by Mashyuf**

### CODE REVIEW

File: wiki/Tournaments/NPC/2/en.md [hunk 1]
```diff
@@ -41,17 +41,19 @@ Generous donations from ![][flag_US] [Nambulance](https://osu.ppy.sh/users/13034
...
+| Streamer | ![][flag_US] [Nambulance](https://osu.ppy.sh/users/13034610), ![][flag_TH] [- Mirage -](https://osu.ppy.sh/users/2298513), ![][flag_CA] [Takiaz](https://osu.ppy.sh/users/11177233), ![][flag_US] [MoopFoopSoup](https://osu.ppy.sh/users/14364661), ![][flag_US] [ClarkyClarker](https://osu.ppy.sh/users/11664411) |
 | Designer | ![][flag_US] [Zeity](https://osu.ppy.sh/users/11367167), ![][flag_TH] [Daitsuku](https://osu.ppy.sh/users/9735525) |
 | Wiki editor | ![][flag_TW] [Mashyuf](https://osu.ppy.sh/users/14356517) |
 
 ## Links
 
 - [Discussion thread](https://osu.ppy.sh/community/forums/topics/1051009)
 - [NPC Discord server](https://discord.gg/Zjb3jR9)
+- [Livestream](https://www.twitch.tv/npc_tournaments)
+- [Challonge bracket](https://challonge.com/wne60mu5)
```
Comment (fajar13k) [inline wiki/Tournaments/NPC/2/en.md:56]:
```suggestion
- [Challonge brackets](https://challonge.com/wne60mu5)
```

File: wiki/Tournaments/NPC/2/en.md [hunk 2]
```diff
@@ -106,6 +108,32 @@ Generous donations from ![][flag_US] [Nambulance](https://osu.ppy.sh/users/13034
...
+  - [Falcom Sound Team J.D.K. - The Fate of The Fairies impossiblexu) \[ignore's Hard\]](https://osu.ppy.sh/beatmapsets/19497#osu/69904)
+- DoubleTime
+  - [FELT - Story (Yohanes) \[Hard\]](https://osu.ppy.sh/beatmapsets/297409#osu/1082285)
+  - [Magnetude - Signals (Firebool) \[m3gB3g's 10 KHz\]](https://osu.ppy.sh/beatmapsets/850972#osu/1810827)
+  - [cYsmix - The Ballad of a Mindless Girl (Mir) \[Chaoz's Hard\]](https://osu.ppy.sh/beatmapsets/792396#osu/1662094)
+- FreeMod
+  - [Silent Siren - Soukai Rock (Shad0w1and) \[Hyper\]](https://osu.ppy.sh/beatmapsets/437797#osu/975628)
+  - [Infected Mushroom - The Legend Of The Black Shawarma (La Cataline) \[Insane\]](https://osu.ppy.sh/beatmapsets/20122#osu/74697)
+- Tiebreaker
+  - **[Ayane - Arrival of Tears (Natsu) \[Insane\]](https://osu.ppy.sh/beatmapsets/321425#osu/796828)
```
Comment (fajar13k) [inline wiki/Tournaments/NPC/2/en.md:135]:
```suggestion
  - **[Ayane - Arrival of Tears (Natsu) \[Insane\]](https://osu.ppy.sh/beatmapsets/321425#osu/796828)**
```

File: wiki/Tournaments/NPC/2/en.md [hunk 3]
```diff
@@ -106,6 +108,32 @@ Generous donations from ![][flag_US] [Nambulance](https://osu.ppy.sh/users/13034
...
 ## Mappools
 
+### Quarterfinals
+
+- NoMod
+  - [Subkulture feat. Klayton of Celldweller - Erasus (ErunamoJAZZ) \[Insane\]](https://osu.ppy.sh/beatmapsets/107979#osu/655216)
+  - [ZUN remixed by LeaF - Resurrection Spell (Muya) \[Hyper\]](https://osu.ppy.sh/beatmapsets/292644#osu/663373)
+  - [Silentroom - Nhelv (Nyxa) \[Rhonen's Hyper\]](https://osu.ppy.sh/beatmapsets/917915#osu/2039004)
+  - [Streetlight Manifesto - The Blonde Lead The Blind (pishifat) \[Insane\]](https://osu.ppy.sh/beatmapsets/376545#osu/826347)
+  - [Ranko - Gensou no Satellite (z1085684963) \[kbsn_zyl's Insane\]](https://osu.ppy.sh/beatmapsets/639995#osu/2298325)
```
Comment (fajar13k) [inline wiki/Tournaments/NPC/2/en.md:118]:
```suggestion
  - [Ranko - Gensou no Satellite (z1085684963) \[kbsn\_zyl's Insane\]](https://osu.ppy.sh/beatmapsets/639995#osu/2298325)
```

File: wiki/Tournaments/NPC/2/en.md [hunk 4]
```diff
@@ -169,6 +197,34 @@ Generous donations from ![][flag_US] [Nambulance](https://osu.ppy.sh/users/13034
 
 ## Match results
 
+### Round of 16
+
+Saturday, 13 June 2020
+
+| Team 1 | | | Team 2 | Match link |
```
Comment (fajar13k) [inline wiki/Tournaments/NPC/2/en.md:204]:
```suggestion
| Team 1 |  |  | Team 2 | Match link |
```
Leave 2 cell padding for empty table head. Apply for others.

File: wiki/Tournaments/NPC/2/en.md [hunk 5]
```diff
@@ -169,6 +197,34 @@ Generous donations from ![][flag_US] [Nambulance](https://osu.ppy.sh/users/13034
...
+| --: | :-: | :-: | :-- | :-- |
+| **Fashionably Late** | **5** | 3 | Oota e-gamers | [#1](https://osu.ppy.sh/community/matches/62858504) |
+| **Click the Circles to the Beat** | **5** | 4 | Degeneracy Gang | [#1](https://osu.ppy.sh/community/matches/62872370) |
+| PoggersFish | 1 | **5** | **Bats from Wuhan** | [#1](https://osu.ppy.sh/community/matches/62876227) |
+| Tablet Slayers | -1 | **0** | **osu!Coin** | Win by default |
```
Comment (fajar13k) [inline wiki/Tournaments/NPC/2/en.md:209]:
```suggestion
| Tablet Slayers | -1 | **0** | **osu!Coin** | *win by default* |
```
lowercase, italicise. Apply to others.

File: wiki/Tournaments/NPC/2/en.md (hunk 1 above)
Comment (cl8n) [reply wiki/Tournaments/NPC/2/en.md:56]: it is only one bracket tho

**Pull request merged**